explainers/hidden-content-explainer.md

```diff
@@ -397,19 +411,11 @@ every time any text is searched for.
 ```
 ##### Pros
 * Simple use cases won't require any script to reveal content.
-* Exposes less information about find-in-page to the page, which improves
-  privacy.
 * Lets the browser handle updating style and scrolling, so there would be no
   need to implement async scrolling to work around use cases with scrolling
   to expanded content.
 ##### Cons
-* Couples the `beforematch` event with `hidden-matchable`. Both of them would
-  need to be shipped together, since `beforematch` would only fire on
-  `hidden-matchable` elements.
-* Doesn't allow for `beforematch` to be used for use cases outside of hidden
-  content, but if it is determined that `beforematch` reveals too much
-  information to the page then this would be necessary.
-  Alternatively, we could still fire `beforematch` on everything we scroll to
-  during find-in-page/ScrollToTextFragment and keep the `hidden-matchable` as
-  an attribute idea which would still keep the Pros of not needing script for
-  a big use case and letting the browser revealing and scrolling.
+* It makes more sense for visibility to be controlled by CSS than an element
+  attribute given that the `display`, `visibility`, and `content-visibility`
+  CSS properties all control visibility.
+* Requires privacy mitigations since the reveal/match is observable by the page.
```

Comment:
were there any other cons for this? I remember talking about it but I don't quite remember everything we said.

Reply:
It's hard to apply this to a lot of content at once (kind of like a css selector or changing one class would let you). That's a bit of a weak point, but that's one of the reasons we prefer css over attribute solutions for things like this.

Reply:
thanks, i added it